### PR TITLE
Fixes addressing the FDOM issue

### PR DESCRIPTION
**Pull request by kathryn-willi**
Here's my current approach for solving the FDOM plotting issue. It seemed to occur within the `add_threshold_lines()` function when there are only NAs in a given week of data. So, I just skip that function in those instances. This is a bit of a stop-gap measure; I think the real issue lies in what the function tries to return when there is no data (i.e., I think one of the conditional statements/parentheses are not quite correct). 

I also made a change to the `summarize_site_parameter_full()` function so that field notes are correctly added to Virridy sites (before this PR, they didn't have field notes).

### CODE REVIEW

File: src/generate_initial_weekly_plots.R [hunk 1]
```diff
@@ -126,17 +126,19 @@ generate_initial_weekly_plots <- function(all_df_list, pending_df_list, site_arg
...
                                            parameter_arg = parameter_arg)
+          }
 
           week_plot <- week_plot +
             theme_bw() +
             scale_x_datetime(date_breaks = "1 day",
                              date_labels = "%b %d",
                              minor_breaks = date_seq,
-                             sec.axis = sec_axis(~., breaks = date_seq, labels = unique(week_plot_data$weekday))) +
+                             sec.axis = sec_axis(~., breaks = date_seq, labels = 1:length(date_seq))) + #unique(week_plot_data$weekday))) +
```
Comment (kathryn-willi) [inline src/generate_initial_weekly_plots.R:141]: @juandlt-csu  Can you confirm that this edit of mine is ok?

Comment (juandlt-csu) [reply src/generate_initial_weekly_plots.R:141]: TBD, have to test it out

File: src/summarize_site_param_full.R
Comment (juandlt-csu) [inline src/summarize_site_param_full.R:1]: I am assuming these updates are unrelated to the FDOM plotting problem.

File: src/add_threshold_lines.R
Comment (juandlt-csu) [inline src/add_threshold_lines.R:1]: nice catch.

File: src/summarize_site_param_full.R
Comment (kathryn-willi) [reply src/summarize_site_param_full.R:1]: correct! This change just allows the field notes to be joined to the lower network's virridy sites

File: src/generate_initial_weekly_plots.R [hunk 2]
```diff
@@ -126,17 +126,19 @@ generate_initial_weekly_plots <- function(all_df_list, pending_df_list, site_arg
             labs(x = "Day",
                  y = "Mean")
 
+          if(!all(is.na(week_plot_data$mean))){
```
Comment (kathryn-willi) [inline src/generate_initial_weekly_plots.R:129]:
```suggestion
         
```

File: src/generate_initial_weekly_plots.R [hunk 3]
```diff
@@ -126,17 +126,19 @@ generate_initial_weekly_plots <- function(all_df_list, pending_df_list, site_arg
...
           week_plot <- add_threshold_lines(plot = week_plot,
                                            plot_data = week_plot_data,
                                            site_arg = site_arg,
                                            parameter_arg = parameter_arg)
+          }
```
Comment (kathryn-willi) [inline src/generate_initial_weekly_plots.R:134]:
```suggestion
 
```

File: src/generate_initial_weekly_plots.R (hunk 1 above)
Comment (kathryn-willi) [inline src/generate_initial_weekly_plots.R:141]:
```suggestion
                             sec.axis = sec_axis(~., breaks = date_seq, labels = unique(week_plot_data$weekday))) +
```
